cmake_templates/zeek-config.in

```diff
@@ -18,6 +21,19 @@ version="@ZEEK_VERSION_FULL@"
 zeek_dist="@ZEEK_DIST@"
 zeekpath="@DEFAULT_ZEEKPATH@"
 
+report_feature() {
+    # $1: test result
+    if [ "$1" = "yes" ] || [ "$1" = "true" ] || [ "$1" = "ON" ]; then
```

Comment:
This does not cover all expressions [CMake considers truthy for booleans](https://cmake.org/cmake/help/latest/command/if.html#constant). Since in CMake this does a case-insensitive compare I'd suggest to upper-case the input and then compare against the full list,

```suggestion
    input=$(echo "$1" | tr '[:lower:]' '[:upper:]')
    if [ "$input" = "1" ] || [ "$input" = "ON" ] || [ "$input" = "YES" ] || [ "$input" = "TRUE" ] || [ "$input" = "Y" ]; then
```

Reply:
Done